Dockerfile

```diff
@@ -0,0 +1,95 @@
+FROM ubuntu:14.04
+MAINTAINER Elico Corp <info@elico-corp.com>
```

Comment:
info@ doesnot exist (this is not an issue it is forwarded.
should we keep it like this or us contact@ or create a docker@